source/features/show-asset-download-count.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import * as api from '../libs/api';
+import * as icons from '../libs/icons';
+import {getOwnerAndRepo} from '../libs/utils';
+import {wrap} from '../libs/dom-utils';
+
+interface Asset {
+	name: string;
+	downloadCount: number;
+}
+interface Tag {
+	[key: string]: Asset[];
+}
+async function getAssetsForTag(tags: string[]): Promise<Tag> {
+	const {ownerName, repoName} = getOwnerAndRepo();
+	const {repository} = await api.v4(`
+		{
+			repository(owner: "${ownerName}", name: "${repoName}") {
+				${tags.map(tag => `
+					${api.escapeKey(tag)}: release(tagName:"${tag}") {
+						releaseAssets(first: 100) {
+							nodes {
+								name
+								downloadCount
+							}
+						}
+					}
+				`)}
+			}
+		}
+	`);
+
+	const assets: Tag = {};
+	for (const [tag, release] of Object.entries(repository)) {
+		assets[tag] = (release as any).releaseAssets.nodes;
+	}
+
+	return assets;
+}
+
+function prettyNumber(value: number): string {
+	if (value <= 999) {
```

Comment:
This doesn't fix it, it actually appears when numbers are under 99. I'd prefer `Number(newValue.toPrecision(3))`